docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.1'
+services:
+  server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    depends_on:
+      - db
+    restart: always
+    ports:
+      - "8080:8080"
+    volumes:
+      - ./app.rsa:/app.rsa
+      - ./app.rsa.pub:/app.rsa.pub
+      - ./cmd/config/config.postgres.json:/cmd/config/config.postgres.json
+  db:
+    image: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: 123455678
```

Comment:
Wrong password: extra 5 is present. 🧮

Reply:
Our password is more secure than the standard 12345678